index.html

```diff
@@ -270,7 +270,28 @@ <h5>LET'S CONNECT</h5>
           <li class="skills-text">Skills</li>
             <img class="disabled-menu2" src="./imgs/disabled-menu/disabled-menu.svg" alt="disabled menu logo" />
       </ul>
+      </section>
+      <!-- Contact Form Section -->
+      <section id="section-contact-form" class="contact-form">
+        <form id="form" class="cform" action="https://formspree.io/f/mrgvnevk">
```

Comment:
- [ ] Kindly ensure that your form has the method attribute with the value 'POST" as this is required as you will be sending data to your formspee. Without this, your form will not be submitted.
e.g
`<form id="form" class="cform" method="POST" action="https://formspree.io/f/mrgvnevk">`

![image](https://github.com/Otromascomotu/My-Portfolio-setup-and-mobile-first/assets/63473584/ca18f842-1d9d-4502-b6e2-dec39c71bfa5)

Reply:
It's done! ✍️✅